AzFilesHybrid/AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4548,16 +4561,19 @@ function Debug-AzStorageAccountADDSAuth {
 
                 $checks["CheckDomainLineOfSight"].Result = "Passed"
                 Write-Verbose "CheckDomainLineOfSight - SUCCESS"
+                Write-TestingPassed
             } catch {
                 $checks["CheckDomainLineOfSight"].Result = "Failed"
                 $checks["CheckDomainLineOfSight"].Issue = $_
-                Write-Error "CheckDomainLineOfSight - FAILED"
-                Write-Error $_
+                Write-TestingFailed -Message $_ -IsUnexpected $true

```

Comment:
I think we may need to make some edits to `Debug-DomainLineOfSight` to also print in the expected way. Maybe we can make it return a bool saying whether we have line-of-sight or not, and then print what we need to here?